util/testUtils.js

```diff
@@ -22,3 +25,30 @@ export const middlewareDispatcher = middleware => (storeData, action) => {
 	return dispatched;
 };
 
+export const parseCookieHeader = (cookieHeader) => {
```

Comment:
nitpick, but i'd like to see consistent function declaration syntax at least within a file? thoughts?